app/assets/v2/js/pages/dashboard.js

```diff
@@ -457,8 +459,9 @@ var refreshBounties = function(event) {
       result.action = result['url'];
       result['title'] = result['title'] ? result['title'] : result['github_url'];
 
+      var project_type = ucwords(result['project_type']) + ' &bull; ';
 
-      result['p'] = ((result['experience_level'] ? result['experience_level'] + ' &bull; ' : ''));
+      result['p'] = project_type + ((result['experience_level'] ? result['experience_level'] : 'Unknown Experience Level') + ' &bull; ');
```

Comment:
Can't we keep it as `' &bull; ' : ''));` -> It looks weird having `'Unknown Experience Level` appearing in on the issues. Looks super odd

Reply:
sure